roaster-restassured/src/main/java/hu/icellmobilsoft/roaster/restassured/response/producer/RestProcessor.java

```diff
@@ -0,0 +1,91 @@
+/*-
+ * #%L
+ * Coffee
+ * %%
+ * Copyright (C) 2020 - 2021 i-Cell Mobilsoft Zrt.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package hu.icellmobilsoft.roaster.restassured.response.producer;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import javax.enterprise.util.AnnotationLiteral;
+import javax.enterprise.util.Nonbinding;
+import javax.inject.Qualifier;
+
+import hu.icellmobilsoft.roaster.restassured.response.producer.impl.ConfigurableResponseProcessor;
+
+/**
+ * CDI qualifier for configuring {@link ConfigurableResponseProcessor}
+ *
+ * @author martin.nagy
+ * @since 0.5.0
+ */
+@Qualifier
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.METHOD, ElementType.FIELD, ElementType.PARAMETER, ElementType.TYPE })
+public @interface RestProcessor {
+
+    /**
+     * Config key of the rest endpoint<br>
+     * ie. if endpoint details are defined in the project-*.yml by the keys: {@code testsuite.example.*=...} then configKey should be
+     * "{@literal testsuite.example}"
+     *
+     * @return config key
+     */
+    @Nonbinding
+    String configKey();
+
+    /**
+     * Expected REST response status code
+     *
+     * @return Expected REST response status code
+     */
+    @Nonbinding
+    int expectedStatusCode() default 200;
+
+    /**
+     * Supports inline instantiation of the {@link RestProcessor} qualifier.
+     */
+    final class Literal extends AnnotationLiteral<RestProcessor> implements RestProcessor {
+        private static final long serialVersionUID = 1L;
+
+        private final String configKey;
+        private final int expectedStatusCode;
```

Comment:
itt jó lenne ha a path paraméter is megadható, úgy akkor nem kellene mindent configba beégetni, használhatóak maradnának a projekteken lévő *Path utvonalak

Reply:
Nem vagyok biztos abban, hogy akarunk ilyet. Ennyi erővel a `baseUriKey`-t is meg lehetne az annotációban adni, és akkor nem kell `configKey` sem. De akár 2 külön qualifier-rel működhet a dolog. @rombow mit gondolsz?

Reply:
Ha bele gondolunk, akkor a path-ok nem valtoznak egy kornyezetben gyakran, de kornyezetenkent igen ha proxy vagy natolas lenne bekapcsolva valahol. A teszt esetek szempontjabol ez valid esetek amire a kodban beegetett path-ok nem kepesek lekezelni. A konfigon keresztul levo path megadasa az mindenkeppen kellene, de ha nincs megadva akkor hasznalhatna az annotacioban felvett erteket.

A jelenlegi megoldas egy irany, amit kesobb itt vagy a hasznalt projektben is boviteni lehet. Biztos vagyok benne hogy idovel bovulni fog a parameter lista.